server/src/routes/food.ts

```diff
@@ -0,0 +1,15 @@
+import { Router } from "express";
+import { create, createMany, deleteFoodById, findAll, findFoodById } from "../controllers/food";
+
+const router = Router();
+
+// router that handles food requests
+router.post("/create", create);
+router.post("/create", createMany);
+router.post("/delete", deleteFoodById);
```

Comment:
**Don't need these**
Why are there two routes with the same name?
Delete should be handled by `router.delete`